src/main/kotlin/graphics/scenery/ui/SwingUiNode.kt

```diff
@@ -0,0 +1,96 @@
+package graphics.scenery.ui
+
+import graphics.scenery.attribute.material.Material
+import graphics.scenery.primitives.Plane
+import graphics.scenery.textures.Texture
+import org.joml.Matrix4f
+import org.joml.Vector2f
+import org.joml.Vector3f
+import org.joml.Vector4f
+
+/**
+ * @author Jan Tiemann <j.tiemann@hzdr.de>
+ * @author Konrad Michel <Konrad.Michel@mailbox.tu-dresden.de>
+ *
+ * Scene representation of a SwingFrame on a plane in form of a texture snapshot render of the frame in combination with SwingBridgeFrame
+ */
+class SwingUiNode(private val swingBridgeFrame : SwingBridgeFrame) : Plane(
```

Comment:
A minor point, but in terms of consistency, `SwingUINode` would make more sense 👍